ocs_ci/ocs/rados_utils.py

```diff
@@ -273,6 +277,76 @@ def get_mgr_proxy_container(self, node, docker_image, proxy_container="mgr_proxy
 
         return mgr_object
 
+    def verify_cephblockpool_status(
+        self,
+        pool_name=constants.DEFAULT_BLOCKPOOL,
+        namespace=None,
+        required_phase=constants.STATUS_READY,
+    ):
+        """
+        Verify the phase of cephblockpool
+
+        Args:
+            pool_name (str): The name of the Ceph block pool
```

Comment:
Please add docstring for function arguments.

Reply:
Addressed in latest commit.